kayenta-core/src/main/java/com/netflix/kayenta/metrics/MetricsRetryConfigurationProperties.java

```diff
@@ -0,0 +1,17 @@
+package com.netflix.kayenta.metrics;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.http.HttpStatus;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Data
+@ConfigurationProperties("kayenta.retry")
```

Comment:
Can you add an example of configuring this to https://github.com/spinnaker/kayenta/blob/master/kayenta-web/config/kayenta.yml

Reply:
yes, of course. I've added default configuration as an example into `kayenta.yml`.
What do you think regarding naming? `kayenta.retry` is ok prefix of better to rename to `kayenta.metrics.retry`?